certification/internal/engine/engine.go

```diff
@@ -293,7 +293,7 @@ func generateBundleHash(bundlePath string) (string, error) {
 		hashBuffer.WriteString(fmt.Sprintf("%s  %s\n", k, files[k]))
 	}
 
-	_, err := artifacts.WriteFile("hashes.txt", hashBuffer.String())
+	_, err := artifacts.WriteFile("hashes.txt", &hashBuffer)
```

Comment:
WriteFile doesn't take a pointer, so how does this work?

Reply:
No, but it takes an interface. And *bytes.Buffer satisfies io.Reader. It has to be a *bytes.Buffer, and not a bytes.Buffer, since the receiver for bytes.Buffer's Read method is a pointer.

Reply:
I forgot io.Reader is an interface.